app/models/invenio_rdm_record_converter.rb

```diff
@@ -127,6 +127,7 @@ def invenio_metadata(gf)
       "contributors": contributors(gf.contributor),
       "dates": gf.date_created.map{ |date| {"date": date, "type": "other", "description": "When the item was originally created."} },
       "languages": gf.language.any?{ |lang| lang.downcase == ENGLISH} ? ["eng"] : "",
+      "identifiers": identifiers(gf.doi.shift),
```

Comment:
Ah! Sorry I thought we were talking about the pids (persistent identifiers) where a managed DOI is stored. That was done above (btw there is a typo in that function name `invennio_pids` -> `invenio_pids`). For this `"identifiers"`, it is pure metadata. That means placing ARK identifiers here would be the right place to do so instead of the DOI. Any alternate identifiers go in this "identifiers" field.

If there are records with multiple DOIs, then the other ones could be placed in "identifiers" too. InvenioRDM only manages one DOI per record. On that note, I noticed the use of [shift](https://ruby-doc.org/core-2.5.0/Array.html#method-i-shift) which seems to mutate the array according to example (I thought `!` was used in Ruby to denote mutation?). How are the tests passing if `invennio_pids` shifts and then identifiers shifts again? Perhaps it is a different `shift` method?

Reply:
> As for alternate identifiers, we can add them here, but what goes into "related identifiers" metadata then?

This is a grey area so I don't think we can go wrong as long as we are consistent. Essentially any identifier that resolves to this record page should go into this `"identifiers"`. Any identifier that is complementary (e.g. has a relationship type like "is related to") or resolves to a record on a different platform should go in `"related_identifiers"`(e.g., pudmed ids and such).  
So we pick a strategy and keep consistent with it and then librarians can tell us if it makes sense to them.